cve_bin_tool/cli.py

```diff
@@ -353,7 +376,9 @@ def main(argv=None, outfile=sys.stdout):
     )
 
     parser.add_argument("-q", "--quiet", action="store_true", help="suppress output")
-
+    parser.add_argument(
+        "-c", "--csv", action="store_true", help="Stores the output in csv file."
```

Comment:
I'm fairly sure we're going to want more than one type of output, so let's make this a `--output` flag or maybe an `--output-type` flag (so you can use `--output` for a filename).  Or maybe make it figure it out from the extension on a filename, defaulting to the usual console output but giving you .csv if your file ends in .csv.   (And eventually, .html, .json, others)